src/app/query-selector/page.module.css

```diff
@@ -0,0 +1,14 @@
+.title {
+  font-size: 40px;
+  font-weight: 600;
+  font-family: 'Inter', sans-serif;
+  margin: 5px 20px;
+  color: #ffffff;
+  display: inline-block;
+  background-color: #3a68ff;
```

Comment:
```suggestion
  background-color: var(--mantine-color-blue-9);
```

Reply:
If you want a dark blue, you can use the mantine blue (9 is on the darker side, a lower number will be lighter). Or if you want the primary theme anchor color (as is the color of the buttons), you can use `var(--mantine-primary-color-filled)`;